environment.yml

```diff
@@ -3,13 +3,13 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - python>=3.8,<3.10
+  - python>=3.9,<3.11
```

Comment:
i've been using `3.11` in my `pypeit` envs for a few weeks now. no issues so i think we can consider it supported. i'd change this to `<3.12`.